arangodb-net-standard/Transport/Http/HttpApiTransport.cs

```diff
@@ -18,34 +18,46 @@ namespace ArangoDBNetStandard.Transport.Http
     /// </remarks>
     public class HttpApiTransport : IApiClientTransport
     {
-        private HttpClient _client;
+        private readonly HttpClient _client;
+        private HttpContentType _contentType;
+
+        private static readonly Dictionary<HttpContentType, string> _contentTypeMap =
+            new Dictionary<HttpContentType, string>
+            {
+                [HttpContentType.Json] = "application/json",
+                [HttpContentType.VPack] = "application/x-velocypack"
```

Comment:
In the VelocyStream readme, I see that in order the content-type for using VelocyPack with HTTP is `application/vpack`. https://github.com/arangodb/velocystream

Where did you see `application/x-velocypack`?

Reply:
I read the velocystream doc as well but `application/vpack` doesn't work.   I found `application/x-velocypack` by looking at the Java driver source.